utils/threadpool/threadpool.h

```diff
@@ -75,7 +77,7 @@ class ThreadPoolGroup
     boost::thread* create_thread(F threadfunc)
     {
         boost::lock_guard<boost::shared_mutex> guard(m);
-        std::auto_ptr<boost::thread> new_thread(new boost::thread(threadfunc));
+        std::unique_ptr<boost::thread> new_thread(new boost::thread(threadfunc));
```

Comment:
This could potentially break SUSE12 build.

Reply:
Couldn't do much about the warning produced in U16. g++ uses 97 standard by default but std::unique_ptr is supported since c++11.

Reply:
the Boost smart ptrs will work.  std::unique_ptr == boost::scoped_ptr